CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ For more information and how-to, see [RFC: Keep A Changelog](https://github.com/
 
 ## [Unreleased]
 
+- Remove the unused inject-v1-pod webhook. [#3885](https://github.com/chaos-mesh/chaos-mesh/pull/3885)
```

Comment:
Move this line into the below `### Removed` section.

Reply:
Done, thanks for pointing out.